src/ui/log_tab.rs

```diff
@@ -642,12 +643,24 @@ impl Component for LogTab<'_> {
                 None => " Log ",
             };
 
+            let log_length: usize = log_lines.len();
             let log_block = Block::bordered()
                 .title(title)
                 .border_type(BorderType::Rounded);
             self.log_height = log_block.inner(chunks[0]).height;
             let log = List::new(log_lines).block(log_block).scroll_padding(7);
             f.render_stateful_widget(log, chunks[0], &mut self.log_list_state);
+
+            // Show scrollbar if more lines than fits in screen height
```

Comment:
```suggestion
            // Show scrollbar if lines don't fit the screen height
```
Also, I'm not sure this comment is even needed because it just describes what the scrollbar is used for